js/dropdown.js

```diff
@@ -527,8 +529,25 @@
      * Place dropdown
      */
     _placeDropdown() {
+      /**
+       * Get closest ancestor that satisfies the condition
+       * @param {Element} el  Element to find ancestors on
+       * @param {Function} condition  Function that given an ancestor element returns true or false
+       * @returns {Element} Return closest ancestor or null if none satisfies the condition
+       */
+      const getClosestAncestor = function(el, condition) {
```

Comment:
Wait, is this a function in a function? Why not use `_placeDropdown` then?

Reply:
Yes, because it is used inside the function only... I dont know how to change the code in an existing PR. Can this be done via the WebInterface?

Reply:
You can push additional commits on the same branch in your repo or use https://github.com/materializecss/materialize/pull/57/files

![grafik](https://user-images.githubusercontent.com/827205/106736691-11dde300-6616-11eb-8dfb-894ee1ae4951.png)